src/id8_i/callbacks/spec_data_file_writer.py

```diff
@@ -65,7 +65,9 @@ def init_specwriter_with_RE(RE):
     """Initialize specwriter with the run engine."""
 
     # make the SPEC file in current working directory (assumes is writable)
-    specwriter.newfile(specwriter.spec_filename)
+    home_path = pathlib.Path.home()
+    new_path = home_path / "/gdata/dm/8ID/8IDI/$cycle_name/$experiment_name/analysis/Scan"
```

Comment:
`home_path` will be ignored here since the next items starts with an absolute path.